### ISSUE TITLE
Shifted quotation mark in <var> elements in generated PDF

### ISSUE THREAD

**Issue by c-kunz** [dita-ot/dita-ot#2711]
**Steps to reproduce**
Render PDF from:
```xml
<synph>
  <kwd>key</kwd><var>"value"</var>
</synph>
```

**Expected result**
In DITA-OT 1.5.4 the generated PDF contains `key"value"`.

**Actual result**
In DITA-OT 2.4.6 the generated PDF contains `key"valu"e`.

**Hint**
The right-most quotation mark *does not shift* when placing a blank right before it.

**Comment by robander** [comment]
It looks like this is an issue in FOP. I can reproduce in 2.4, 2.5, and 3.0 when using FOP; but when I use Antenna House the output is correct. We've upgraded FOP a couple times since 1.5 and I'm guessing this is a regression in that tool. The FO markup sent into FOP is correct (I'm adding newlines so it is easier to read here):
```
<fo:inline>
<fo:inline line-height-shift-adjustment="disregard-shifts" 
   font-family="Courier New, Courier, Arial Unicode MS, Tahoma, Batang, SimSun">key</fo:inline>
<fo:inline font-style="italic">"value"</fo:inline>
</fo:inline>
```

**Comment by robander** [comment]
I can even reproduce this with just italic markup in a paragraph:
`<i>"value"</i>`

But ... @simalot has just tried building that same `topic.fo` file with FOP outside of the context of DITA-OT, and cannot reproduce the problem. So I'm less certain what is going on now.

**Comment by robander** [comment]
Here was my full test topic, published on its own to PDF: 

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN"  "topic.dtd">
<topic id="synph" xml:lang="en-us">
<title>Test var in synph</title>
<shortdesc>This is a test of <synph><kwd>key</kwd><var>"value"</var></synph> inside
shortdesc</shortdesc>
<body>
<p>Now I'm going to test it in a paragraph: <synph><kwd>key</kwd><var>"value"</var></synph> and see what happens</p>
<p>Now I'm going to tweak it in a paragraph: <synph><var>"value"</var></synph> and see what
happens</p>
<p>Now I'm going to tweak use just italic: <i>"value"</i> and see what happens</p>
<p>Now I'm going to test it with kwd in a paragraph: <synph><kwd>key</kwd><kwd>"value"</kwd></synph> and see what happens</p>
<p>Now I'm going to test it with codeph in a paragraph:
<synph><kwd>key</kwd><codeph>"value"</codeph></synph> and see what happens</p>
</body>
</topic>
```

**Comment by robander** [comment]
Marking this closed - I've retested with DITA-OT 3.4, which updated the version of FOP, and I'm no longer able to reproduce the issue.
